addon/enums/header-menu.js

```diff
@@ -0,0 +1,16 @@
+import Type from './types';
+
+export default Type.create({
+    ACCOUNT: {
```

Comment:
This seems more like a const local to the header-menu.

Move this to the component instead of being an enum.

Also this options should come from whomever is calling the component instead of having them hardcoded. That will allow to abstract the component implementation